test/funcfile-docker-rt-tests/fn_test.go

```diff
@@ -0,0 +1,165 @@
+package tests
+
+import (
+        "bytes"
+	"fmt"
+	"io/ioutil"
+        "log"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+
+
+func TestDockerRuntime(t *testing.T) {
+        testname := "TestDockerRuntime"
+        testfiles := []string{ "Dockerfile", "func.go" }
+
+        currdir, err := os.Getwd() 
+	if err != nil {
+		t.Fatalf("ERROR: Failed to get current directory: err: %s", err)
+	}
+	testdir, err := ioutil.TempDir("", "cli_test_funcfile-docker-rt_" + testname)
+	if err != nil {
+		t.Fatalf("ERROR: Failed to make tmp test directory: err: %s", err)
+	}
+        defer cleanup(t, currdir, testdir)
+	fnTestBin := setupTestFiles(t, testname, currdir, testdir, testfiles)   
+	runFnInit(t, testname, fnTestBin)
+	runFnBuild(t, testname, fnTestBin)
+	runFnRun(t, testname, fnTestBin)
+}
+
+func TestDockerRuntimeNoDockerfile(t *testing.T) {
+        testname := "TestDockerRuntimeNoDockerfile"
+        testfiles := []string{ "func.yaml", "func.go" }
+
+        currdir, err := os.Getwd() 
+	if err != nil {
+		t.Fatalf("ERROR: Failed to get current directory: err: %s", err)
```

Comment:
err.Error()

Reply:
Or %v instead of %s.

Reply:
That line follows existing cli/main_test.go on formatting 'err' in t.Fatalf().  agree with @treeder %v is proper. changed